application/core/plugins/AuthLDAP/AuthLDAP.php

```diff
@@ -117,6 +117,14 @@ class AuthLDAP extends LimeSurvey\PluginManager\AuthPluginBase
         'allowInitialUser' => array(
             'type' => 'checkbox',
             'label' => 'Allow initial user to login via LDAP',
+        ),
+        // Need to be defined here otherwise saving does not work
+        'defaultRole' => array(
+            'type' => 'select',
+            'options' => array('none'=>'None'),
```

Comment:
Empty option with

```
'htmlOptions' => [
    'empty' => 'None'
]
```

default is empty.

Think it's better

Reply:
I agree and updatet the pull request